iOSNativeSwiftEncryptedNotificationTemplate/Podfile

```diff
@@ -0,0 +1,29 @@
+platform :ios, '12.2'
+
+project 'EncryptedNotificationTemplate.xcodeproj'
+target 'EncryptedNotificationTemplate' do
+    source 'https://cdn.cocoapods.org/'
+    use_frameworks!
+
+    pod 'SalesforceSDKCommon', :path => 'mobile_sdk/SalesforceMobileSDK-iOS'
```

Comment:
You don't really need these 4 entries - `MobileSync` should transitively pull the others in through its `Podfile`.

Reply:
We do because we want them pulled from a local path.
Otherwise, it will try to pull to one published.

Reply:
Ah yes, I forgot that the local ones aren't updated to pull transitively locally as well.

Reply:
Our libraries happen to live in one directory but it's typically not the case. So there would be no way for cocoapod to know what local pod of B to get when using a local pod of A (assuming A depends on B) or even to know if that's what one wants.

Reply:
Makes sense.